Objects/EventItem/EventItem.gd

```diff
@@ -1,25 +1,31 @@
 extends Node2D
 
+const TRACK_HEIGHT = 260
 export(int) var gold_reward = 65
 export(float) var move_speed = -100
+onready var cam_pos = get_node("/root/World").find_node("Camera2D").position
 
 func _ready():
 	var value = randi()%800+160
 	set_start_position(value,293)
 
-func _process(delta):
-	move(delta)
-	
-func move(delta):
-	position.x += move_speed * delta
-
 func set_start_position(x, y):
-	position.x = x
-	position.y = y
+	if "DwarfInBalloon" in self.get_name(): #other position for DwarfInBallon
```

Comment:
This should be done in inherited script DwarfInBaloon.gd, by changing start position values.